lib/ex_doc/translation.ex

```diff
@@ -0,0 +1,569 @@
+
+  def t(@locale_spanish_international, "Keyboard Shortcuts"),
+    do: "Combinaciones de Teclas"
+
+  def t(@locale_spanish_international, "Link"), do: "Enlace"
+  def t(@locale_spanish_international, "Link to this"), do: "Enlace a esto"
+
+  def t(@locale_spanish_international, "Link to this callback"),
+    do: "Enlace a esta devolución de llamada"
+
+  def t(@locale_spanish_international, "Link to this function"),
+    do: "Enlace a esta función"
+
+  def t(@locale_spanish_international, "Link to this section"),
+    do: "Enlace a esta sección"
+
+  def t(@locale_spanish_international, "Link to this type"),
+    do: "Enlace a este tipo"
+
+  def t(@locale_spanish_international, "Mix Tasks"), do: "Tareas de Mix"
+  def t(@locale_spanish_international, "Modules"), do: "Módulos"
+
+  def t(@locale_spanish_international, "Multiple words (such as"),
+    do: "Palabras múltiples (tales como"
+
+  def t(@locale_spanish_international, "Option"), do: "Opción"
+  def t(@locale_spanish_international, "Options"), do: "Opciones"
+  def t(@locale_spanish_international, "Page not found"), do: "Página no encontrada"
+  def t(@locale_spanish_international, "Pages"), do: "Páginas"
+  def t(@locale_spanish_international, "Search"), do: "Buscar"
+  def t(@locale_spanish_international, "Search..."), do: "Buscar..."
+
+  def t(@locale_spanish_international, "Search results for"),
+    do: "Resultados de la búsqueda para"
+
+  def t(@locale_spanish_international, "Search the documentation"),
+    do: "Buscar en la documentación"
+
+  def t(
+        @locale_spanish_international,
+        "Sorry, but the page you were trying to get to, does not exist."
+      ),
+      do: "Lo siento, pero la página a la que estás intentando acceder, no existe."
+
+  def t(@locale_spanish_international, "Sorry, we couldn't find anything for"),
+    do: "Lo sentimos, no se pudo encontrar nada para"
+
+  def t(@locale_spanish_international, "Summary"), do: "Resumen"
+  def t(@locale_spanish_international, "Table of contents"), do: "Tabla de contenidos"
+  def t(@locale_spanish_international, "This"), do: "Este"
+
+  def t(@locale_spanish_international, "This callback is deprecated"),
+    do: "Esta devolución de llamada quedó obsoleta"
+
+  def t(@locale_spanish_international, "This is deprecated"), do: "Esto quedó obsoleto"
+
+  def t(@locale_spanish_international, "This function is deprecated"),
+    do: "Esta función quedó obsoleta"
+
+  def t(@locale_spanish_international, "This type is deprecated"), do: "Este tipo quedó obsoleto"
+
+  def t(
+        @locale_spanish_international,
+        "To quickly go to a module, type, or function, use the autocompletion feature in the sidebar search."
+      ),
+      do:
+        "Para rápidamente acceder a un módulo, tipo o función, usa la característica de autocompletado en la búsqueda de la barra lateral"
+
+  def t(@locale_spanish_international, "Toggle night mode"), do: "Conmutar modo noche"
```

Comment:
"Cambiar a" sound better for "night mode", but it is not the right fit for "Toggle sidebar"